goose_test.go

```diff
@@ -52,6 +85,42 @@ func TestDefaultBinary(t *testing.T) {
 	}
 }
 
+func TestIssue532(t *testing.T) {
+	t.Parallel()
+
+	migrationsDir := filepath.Join("examples", "sql-migrations")
+	count := countSQLFiles(t, migrationsDir)
+	check.Number(t, count, 3)
+
+	tempDir := t.TempDir()
+	dirFlag := "--dir=" + migrationsDir
+	params := []string{dirFlag, "sqlite3", filepath.Join(tempDir, "sql.db")}
+
+	tt := []struct {
+		command   string
+		skipCheck bool
+		output    string
+	}{
+		{"up", true, ""},
+		{"up", false, "goose: no migrations to run. current version: 3"},
```

Comment:
This test covers the wrong version in the print line. It's also intended to be an improved way to write these kinds of tests. We'll likely want to add "golden files" to capture the output. 

Note, the binary is built once and reused so future tests can just call `runGoose`